### DISCUSSION TITLE
Does Orange C support `__attribute__((weak))`?

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#961] [General]
This is from GCC/Clang.

**Comment by GitMensch** [comment]
Please use the discussions for general questions (and in your case: instead of closing and opening a new for the same topic: just edit them)

**Reply by ghost** [reply]
From the table posted by @chuggafan, I know `__attribute__((weak))` is currently not supported.

https://github.com/LADSoft/OrangeC/discussions/960#discussioncomment-7223974

So I created an issue asking if you will support it. Is that wrong? The discussion thread is a question about whether you supported it or not. I already got the answer (which is not supported). Why do I have to keep it open? Feature request is for issues, not discussions. Please don't do nitpicking.

**Reply by GitMensch** [reply]
Nope, closing the discussion is fine. The notifications that GitHub sent out looked like you've created an issue, then closed it, then created a new one.
Nitpicking: your new issues are questions (which go to discussions ;-), it is fine to have this as a feature request to add it (at least parse and ignore the attribute) [one for both likely would have been enough], please edit the issue (possibly changing its title from a question to a feature request) and add an example along with occ's current output - this way everyone can easily reproduce that and use the test file in a later test case.

Thank you!

**Reply by ghost** [reply]
OK. Didn't see your message.
